### PR TITLE
fix: swp file was watched and copy to dist

### PR DESCRIPTION
**Pull request by sabakugaara**

### CODE REVIEW

File: packages/ruo-cli/src/gulpfile.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ gulp.task('build-watch', ['cached-build'], () => {
       let ext = path.extname(vinyl.path)
       if (ext === '.js') {
         compile(vinyl.path, 'change')
-      } else {
+      } else if (ext === '.yaml') {
```
Comment (sabakugaara) [inline packages/ruo-cli/src/gulpfile.js]: `rc.source` 目录下，目前应该只有 `.js|.yaml` 两种类型文件

Comment (CatTail) [inline packages/ruo-cli/src/gulpfile.js]: No, there will be other assets.

Comment (CatTail) [inline packages/ruo-cli/src/gulpfile.js]: You can update to new version an run the tests, it should fail.

Comment (sabakugaara) [inline packages/ruo-cli/src/gulpfile.js]: all passed

Comment (CatTail) [inline packages/ruo-cli/src/gulpfile.js]: No, I don't mean this proejct, I mean UpYUN internal API project.

**Pull request merged**